package.json

```diff
@@ -1808,16 +1808,11 @@
           "description": "%cmake-tools.configuration.cmake.exportCompileCommandsFile.description%"
         },
         "cmake.useCMakePresets":  {
-          "oneOf": [
-            {
-              "type": "boolean"
-            },
-            {
-              "type": "string",
-              "enum": [
-                "automatic"
-              ]
-            }
+          "type": "string",
+          "enum": [
+            "always",
+            "never",
+            "automatic"
```

Comment:
Erika's documentation shortens it to 'auto'.  Can you rename it?

Reply:
CMakeCommunicationMode also have an 'automatic' option, and I want to make them consistent.

Reply:
We also have:
```
"cmake.outputLogEncoding": {
          "type": "string",
          "default": "auto",
          "description": "%cmake-tools.configuration.cmake.outputLogEncoding.description%",
          "scope": "window"
        },
```

So it seems we're not 100% consistent right now already.